flybywiresim/a32nx
# feat(eicas): Show ISA DEV on EICAS

Fixes #5029 

## Summary of Changes
The ISA DEV is now shown on the EICAS if the ADIRS are aligned and the altimeter knob is set to STD. This is computed by:
```
ISA = 15 - (AltStd * 1.98 / 1000)
delta ISA = SAT - ISA
```
where AltStd is the altimeter reading while barometer is set to Std. The international standard atmosphere (ISA) model is expected to have a temperature of 15 °C at sea level and decrease by about 1.98 °C / 1000 ft up to the tropopause. At which point it stays at -56.5 °C. The EICAS shows the deviation of the current actual air temperature to the temperature predicted by ISA. Thank you to @lukecologne and @donstim for explaining!

Furthermore, I right-aligned all temperature values.

## Screenshots (if necessary)
Before: (still what you see if altimeter is not set to std)
![aligned-not-std](https://user-images.githubusercontent.com/22713769/121262286-27a93080-c8b4-11eb-8a89-366b34332541.png)
After:
![right-aligned-std](https://user-images.githubusercontent.com/22713769/121505009-bddb7480-c9e2-11eb-977b-73dad707c6fd.png)

## References
- FCOM
- Discord user Illuminati#0055 (type rated on the A320) opened the PR which should validate this logic




## Additional context



Discord username (if different from GitHub): BlueberryKing#6641

## Testing instructions
- Spawn in cold and dark, power up the airplane without aligning the ADIRS. SAT and TAT should be shown in amber XX. ISA is not visible.
- Align ADIRS and make sure ISA is only shown if altimeter setting is STD.
- Ensure all temperature values are right-aligned, meaning they should all have the same spacing to the °C symbol.
- Change ISA in the ingame weather settings. Changes should be reflected on the EICAS.
- Take off and climb up to about FL 380 (EXP CLB is helpful for this). ISA should stay about the same during the whole climb while SAT and TAT decrease.
- Ensure in particular that ISA does not change once passing tropopause at 36089 feet. SAT and TAT should stop decreasing too.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
